OrbitLinuxTracing/PerfEventProcessor.h

```diff
@@ -78,19 +78,22 @@ class PerfEventProcessor {
 
   void ClearVisitors() { visitors_.clear(); }
 
+  void SetDiscardedOutOfOrderCounter(
```

Comment:
I think this needs a comment why this counter is passed as a shared pointer.

Reply:
I would rather just pass a bare pointer here.

Reply:
Thanks for changing that. From my side, looks good :)